eks-distro-base/Makefile

```diff
@@ -87,3 +90,12 @@ build: local-images
 
 .PHONY: release
 release: images
+
+.PHONY: update
+update: 
+	$(eval RETURN_VALUE="$(shell ./check_update.sh $(IMAGE_REPO) $(IMAGE_NAME) $(IMAGE_TAG))")
```

Comment:
Are the eval and shell keywords required here? I forget if I asked you already, but I though it might just work?